### ISSUE TITLE
Adding logo to the other sections of the page

### ISSUE THREAD

**Issue by Anikinee** [Anupkjha2601/finalproj#26]
In the navigation bar, when the other sections such as 'About Us' or 'Gallery' are clicked it does not show the logo. A logo must be added to all the sections. I can do it please assign it to me @Anupkjha2601 Sir.

**Comment by ydv-ankit** [comment]
I'm working on it

**Comment by adityavpawar** [comment]
Please assign me this issue so that I can resolve it under GSSOC-23

**Comment by Anupkjha2601** [comment]
yes @[ydv-ankit](https://github.com/ydv-ankit) ydv-ankit he is working on this issue , you can find other .

**Comment by ydv-ankit** [comment]
PR raised: #159
